vita3k/gui/src/user_management.cpp

```diff
@@ -689,8 +686,9 @@ void draw_user_management(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::TextColored(GUI_COLOR_TEXT, "%s", lang["name"].c_str());
         ImGui::SetCursorPos(INPUT_NAME_POS);
         ImGui::PushItemWidth(INPUT_NAME_SIZE);
-        if (ImGui::InputText("##user_name", temp.name.data(), SCE_NP_ONLINEID_MAX_LENGTH))
-            temp.name = temp.name.data();
+        char user_name_buf[SCE_NP_ONLINEID_MAX_LENGTH + 1]{};
+        if (ImGui::InputText("##user_name", user_name_buf, sizeof(user_name_buf)))
+            temp.name = user_name_buf;
```

Comment:
```suggestion
        char user_name_buf[SCE_NP_ONLINEID_MAX_LENGTH + 1]{};
        strncpy(user_name_buf, temp.name.c_str(), SCE_NP_ONLINEID_MAX_LENGTH);
        if (ImGui::InputText("##user_name", user_name_buf, SCE_NP_ONLINEID_MAX_LENGTH))
            temp.name = user_name_buf;
```
iIf you really want using your code, change your code not worksq by this, tested and works fine 👌